doc/audio.md

```diff
@@ -28,7 +30,7 @@ This uses the [audioplayers](https://github.com/luanpotter/audioplayer) lib, in
 If you want to play indefinitely, just use `loop` function:
 
 ```dart
-    Flame.audio.loop('music.mp3');
+    Flame.audio.loopMusic('music.mp3');
```

Comment:
Is this using the low latency api, right? Wouldn't it be better to use `loopSfx`?

Because it will have the same limits of the `playSfx` in terms of file size...

Reply:
also using loopMusic won't fix the gap between iterations, now that I think about it...